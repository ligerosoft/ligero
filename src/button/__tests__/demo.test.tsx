import demoTest from '../../../tests/demo';

demoTest('button');
