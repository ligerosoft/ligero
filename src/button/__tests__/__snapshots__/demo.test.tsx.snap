// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button demos render ./src/button/demos/basic.tsx correctly 1`] = `
Array [
  "按钮类型：",
  <button
    class="ligero-button ligero-button--primary"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        提示按钮
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--default"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        默认按钮
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--success"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        成功按钮
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--warning"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        警告按钮
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--danger"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        危险按钮
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--dashed"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        虚线按钮
      </span>
    </div>
  </button>,
  <div>
    禁用类型：
  </div>,
  <button
    class="ligero-button ligero-button--default ligero-button--disabled"
    disabled=""
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        提示按钮
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--success ligero-button--disabled"
    disabled=""
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        成功按钮
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--warning ligero-button--disabled"
    disabled=""
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        警告按钮
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--danger ligero-button--disabled"
    disabled=""
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        危险按钮
      </span>
    </div>
  </button>,
  <div>
    朴素按钮
  </div>,
  <button
    class="ligero-button ligero-button--default ligero-button--plain"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        提示按钮
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--primary ligero-button--plain"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        提示按钮
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--success ligero-button--plain"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        成功按钮
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--warning ligero-button--plain"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        警告按钮
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--danger ligero-button--plain"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        危险按钮
      </span>
    </div>
  </button>,
  <div>
    按钮自定义颜色
  </div>,
  <button
    class="ligero-button ligero-button--default ligero-button--plain"
    style="background: rgb(255, 255, 255); border-color: purple; color: purple;"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        朴素单色
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--default"
    style="background: purple; border-color: purple; color: rgb(255, 255, 255);"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        单色
      </span>
    </div>
  </button>,
  <div>
    按钮自定义形状
  </div>,
  <button
    class="ligero-button ligero-button--primary ligero-button--square"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        方形
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--primary ligero-button--round"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        圆形
      </span>
    </div>
  </button>,
  <div>
    按钮自定义大小
  </div>,
  <button
    class="ligero-button ligero-button--primary ligero-button--xs"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        迷你按钮
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--primary ligero-button--sm"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        小号按钮
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--primary ligero-button--md"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        普通按钮
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--primary ligero-button--lg"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        大号按钮
      </span>
    </div>
  </button>,
  <div>
    跳转
  </div>,
  <a
    class="ligero-button ligero-button--primary"
    href="https://github.com/ligerosoft/ligero"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        跳转
      </span>
    </div>
  </a>,
  <button
    class="ligero-button ligero-button--primary"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__loading"
      >
        <span
          aria-label="loading"
          class="anticon anticon-loading anticon-spin"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="loading"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="0 0 1024 1024"
            width="1em"
          >
            <path
              d="M988 548c-19.9 0-36-16.1-36-36 0-59.4-11.6-117-34.6-171.3a440.45 440.45 0 00-94.3-139.9 437.71 437.71 0 00-139.9-94.3C629 83.6 571.4 72 512 72c-19.9 0-36-16.1-36-36s16.1-36 36-36c69.1 0 136.2 13.5 199.3 40.3C772.3 66 827 103 874 150c47 47 83.9 101.8 109.7 162.7 26.7 63.1 40.2 130.2 40.2 199.3.1 19.9-16 36-35.9 36z"
            />
          </svg>
        </span>
      </span>
      <span
        class="ligero-button__text"
      >
        loading
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--default ligero-button--plain"
    style="background: rgb(255, 255, 255); border-color: purple; color: purple;"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__loading"
      >
        <span
          aria-label="loading"
          class="anticon anticon-loading anticon-spin"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="loading"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="0 0 1024 1024"
            width="1em"
          >
            <path
              d="M988 548c-19.9 0-36-16.1-36-36 0-59.4-11.6-117-34.6-171.3a440.45 440.45 0 00-94.3-139.9 437.71 437.71 0 00-139.9-94.3C629 83.6 571.4 72 512 72c-19.9 0-36-16.1-36-36s16.1-36 36-36c69.1 0 136.2 13.5 199.3 40.3C772.3 66 827 103 874 150c47 47 83.9 101.8 109.7 162.7 26.7 63.1 40.2 130.2 40.2 199.3.1 19.9-16 36-35.9 36z"
            />
          </svg>
        </span>
      </span>
      <span
        class="ligero-button__text"
      >
        单色
      </span>
    </div>
  </button>,
  <div>
    带icon的button
  </div>,
  <button
    class="ligero-button ligero-button--primary"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__icon"
      >
        <span
          aria-label="like"
          class="anticon anticon-like"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="like"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M885.9 533.7c16.8-22.2 26.1-49.4 26.1-77.7 0-44.9-25.1-87.4-65.5-111.1a67.67 67.67 0 00-34.3-9.3H572.4l6-122.9c1.4-29.7-9.1-57.9-29.5-79.4A106.62 106.62 0 00471 99.9c-52 0-98 35-111.8 85.1l-85.9 311H144c-17.7 0-32 14.3-32 32v364c0 17.7 14.3 32 32 32h601.3c9.2 0 18.2-1.8 26.5-5.4 47.6-20.3 78.3-66.8 78.3-118.4 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7-.2-12.6-2-25.1-5.6-37.1zM184 852V568h81v284h-81zm636.4-353l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 22.4-13.2 42.6-33.6 51.8H329V564.8l99.5-360.5a44.1 44.1 0 0142.2-32.3c7.6 0 15.1 2.2 21.1 6.7 9.9 7.4 15.2 18.6 14.6 30.5l-9.6 198.4h314.4C829 418.5 840 436.9 840 456c0 16.5-7.2 32.1-19.6 43z"
            />
          </svg>
        </span>
      </span>
      <span
        class="ligero-button__text"
      >
        loading
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--primary ligero-button--block"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__icon"
      >
        <span
          aria-label="like"
          class="anticon anticon-like"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="like"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M885.9 533.7c16.8-22.2 26.1-49.4 26.1-77.7 0-44.9-25.1-87.4-65.5-111.1a67.67 67.67 0 00-34.3-9.3H572.4l6-122.9c1.4-29.7-9.1-57.9-29.5-79.4A106.62 106.62 0 00471 99.9c-52 0-98 35-111.8 85.1l-85.9 311H144c-17.7 0-32 14.3-32 32v364c0 17.7 14.3 32 32 32h601.3c9.2 0 18.2-1.8 26.5-5.4 47.6-20.3 78.3-66.8 78.3-118.4 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7-.2-12.6-2-25.1-5.6-37.1zM184 852V568h81v284h-81zm636.4-353l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 22.4-13.2 42.6-33.6 51.8H329V564.8l99.5-360.5a44.1 44.1 0 0142.2-32.3c7.6 0 15.1 2.2 21.1 6.7 9.9 7.4 15.2 18.6 14.6 30.5l-9.6 198.4h314.4C829 418.5 840 436.9 840 456c0 16.5-7.2 32.1-19.6 43z"
            />
          </svg>
        </span>
      </span>
      <span
        class="ligero-button__text"
      >
        loading
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--primary ligero-button--only-icon"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__icon"
      >
        <span
          aria-label="like"
          class="anticon anticon-like"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="like"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M885.9 533.7c16.8-22.2 26.1-49.4 26.1-77.7 0-44.9-25.1-87.4-65.5-111.1a67.67 67.67 0 00-34.3-9.3H572.4l6-122.9c1.4-29.7-9.1-57.9-29.5-79.4A106.62 106.62 0 00471 99.9c-52 0-98 35-111.8 85.1l-85.9 311H144c-17.7 0-32 14.3-32 32v364c0 17.7 14.3 32 32 32h601.3c9.2 0 18.2-1.8 26.5-5.4 47.6-20.3 78.3-66.8 78.3-118.4 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7-.2-12.6-2-25.1-5.6-37.1zM184 852V568h81v284h-81zm636.4-353l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 22.4-13.2 42.6-33.6 51.8H329V564.8l99.5-360.5a44.1 44.1 0 0142.2-32.3c7.6 0 15.1 2.2 21.1 6.7 9.9 7.4 15.2 18.6 14.6 30.5l-9.6 198.4h314.4C829 418.5 840 436.9 840 456c0 16.5-7.2 32.1-19.6 43z"
            />
          </svg>
        </span>
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--primary ligero-button--round ligero-button--only-icon"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__icon"
      >
        <span
          aria-label="like"
          class="anticon anticon-like"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="like"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M885.9 533.7c16.8-22.2 26.1-49.4 26.1-77.7 0-44.9-25.1-87.4-65.5-111.1a67.67 67.67 0 00-34.3-9.3H572.4l6-122.9c1.4-29.7-9.1-57.9-29.5-79.4A106.62 106.62 0 00471 99.9c-52 0-98 35-111.8 85.1l-85.9 311H144c-17.7 0-32 14.3-32 32v364c0 17.7 14.3 32 32 32h601.3c9.2 0 18.2-1.8 26.5-5.4 47.6-20.3 78.3-66.8 78.3-118.4 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7-.2-12.6-2-25.1-5.6-37.1zM184 852V568h81v284h-81zm636.4-353l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 22.4-13.2 42.6-33.6 51.8H329V564.8l99.5-360.5a44.1 44.1 0 0142.2-32.3c7.6 0 15.1 2.2 21.1 6.7 9.9 7.4 15.2 18.6 14.6 30.5l-9.6 198.4h314.4C829 418.5 840 436.9 840 456c0 16.5-7.2 32.1-19.6 43z"
            />
          </svg>
        </span>
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--primary ligero-button--round ligero-button--xs ligero-button--only-icon"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__icon"
      >
        <span
          aria-label="like"
          class="anticon anticon-like"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="like"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M885.9 533.7c16.8-22.2 26.1-49.4 26.1-77.7 0-44.9-25.1-87.4-65.5-111.1a67.67 67.67 0 00-34.3-9.3H572.4l6-122.9c1.4-29.7-9.1-57.9-29.5-79.4A106.62 106.62 0 00471 99.9c-52 0-98 35-111.8 85.1l-85.9 311H144c-17.7 0-32 14.3-32 32v364c0 17.7 14.3 32 32 32h601.3c9.2 0 18.2-1.8 26.5-5.4 47.6-20.3 78.3-66.8 78.3-118.4 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7-.2-12.6-2-25.1-5.6-37.1zM184 852V568h81v284h-81zm636.4-353l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 22.4-13.2 42.6-33.6 51.8H329V564.8l99.5-360.5a44.1 44.1 0 0142.2-32.3c7.6 0 15.1 2.2 21.1 6.7 9.9 7.4 15.2 18.6 14.6 30.5l-9.6 198.4h314.4C829 418.5 840 436.9 840 456c0 16.5-7.2 32.1-19.6 43z"
            />
          </svg>
        </span>
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--primary ligero-button--round ligero-button--sm ligero-button--only-icon"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__icon"
      >
        <span
          aria-label="like"
          class="anticon anticon-like"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="like"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M885.9 533.7c16.8-22.2 26.1-49.4 26.1-77.7 0-44.9-25.1-87.4-65.5-111.1a67.67 67.67 0 00-34.3-9.3H572.4l6-122.9c1.4-29.7-9.1-57.9-29.5-79.4A106.62 106.62 0 00471 99.9c-52 0-98 35-111.8 85.1l-85.9 311H144c-17.7 0-32 14.3-32 32v364c0 17.7 14.3 32 32 32h601.3c9.2 0 18.2-1.8 26.5-5.4 47.6-20.3 78.3-66.8 78.3-118.4 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7-.2-12.6-2-25.1-5.6-37.1zM184 852V568h81v284h-81zm636.4-353l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 22.4-13.2 42.6-33.6 51.8H329V564.8l99.5-360.5a44.1 44.1 0 0142.2-32.3c7.6 0 15.1 2.2 21.1 6.7 9.9 7.4 15.2 18.6 14.6 30.5l-9.6 198.4h314.4C829 418.5 840 436.9 840 456c0 16.5-7.2 32.1-19.6 43z"
            />
          </svg>
        </span>
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--primary ligero-button--round ligero-button--md ligero-button--only-icon"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__icon"
      >
        <span
          aria-label="like"
          class="anticon anticon-like"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="like"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M885.9 533.7c16.8-22.2 26.1-49.4 26.1-77.7 0-44.9-25.1-87.4-65.5-111.1a67.67 67.67 0 00-34.3-9.3H572.4l6-122.9c1.4-29.7-9.1-57.9-29.5-79.4A106.62 106.62 0 00471 99.9c-52 0-98 35-111.8 85.1l-85.9 311H144c-17.7 0-32 14.3-32 32v364c0 17.7 14.3 32 32 32h601.3c9.2 0 18.2-1.8 26.5-5.4 47.6-20.3 78.3-66.8 78.3-118.4 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7-.2-12.6-2-25.1-5.6-37.1zM184 852V568h81v284h-81zm636.4-353l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 22.4-13.2 42.6-33.6 51.8H329V564.8l99.5-360.5a44.1 44.1 0 0142.2-32.3c7.6 0 15.1 2.2 21.1 6.7 9.9 7.4 15.2 18.6 14.6 30.5l-9.6 198.4h314.4C829 418.5 840 436.9 840 456c0 16.5-7.2 32.1-19.6 43z"
            />
          </svg>
        </span>
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--primary ligero-button--round ligero-button--lg ligero-button--only-icon"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__icon"
      >
        <span
          aria-label="like"
          class="anticon anticon-like"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="like"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M885.9 533.7c16.8-22.2 26.1-49.4 26.1-77.7 0-44.9-25.1-87.4-65.5-111.1a67.67 67.67 0 00-34.3-9.3H572.4l6-122.9c1.4-29.7-9.1-57.9-29.5-79.4A106.62 106.62 0 00471 99.9c-52 0-98 35-111.8 85.1l-85.9 311H144c-17.7 0-32 14.3-32 32v364c0 17.7 14.3 32 32 32h601.3c9.2 0 18.2-1.8 26.5-5.4 47.6-20.3 78.3-66.8 78.3-118.4 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7 0-12.6-1.8-25-5.4-37 16.8-22.2 26.1-49.4 26.1-77.7-.2-12.6-2-25.1-5.6-37.1zM184 852V568h81v284h-81zm636.4-353l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 16.5-7.2 32.2-19.6 43l-21.9 19 13.9 25.4a56.2 56.2 0 016.9 27.3c0 22.4-13.2 42.6-33.6 51.8H329V564.8l99.5-360.5a44.1 44.1 0 0142.2-32.3c7.6 0 15.1 2.2 21.1 6.7 9.9 7.4 15.2 18.6 14.6 30.5l-9.6 198.4h314.4C829 418.5 840 436.9 840 456c0 16.5-7.2 32.1-19.6 43z"
            />
          </svg>
        </span>
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--primary ligero-button--round ligero-button--lg"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        A
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--primary ligero-button--round ligero-button--md"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        A
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--primary ligero-button--round ligero-button--sm"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        A
      </span>
    </div>
  </button>,
  <button
    class="ligero-button ligero-button--primary ligero-button--round ligero-button--xs"
  >
    <div
      class="ligero-button__content"
    >
      <span
        class="ligero-button__text"
      >
        A
      </span>
    </div>
  </button>,
]
`;
